pkg/controller/datavolume/import-controller_test.go

```diff
@@ -267,6 +268,7 @@ var _ = Describe("All DataVolume Tests", func() {
 			Expect(pvc.Labels[LabelDefaultInstancetypeKind]).To(Equal(LabelDefaultInstancetypeKind))
 			Expect(pvc.Labels[LabelDefaultPreference]).To(Equal(LabelDefaultPreference))
 			Expect(pvc.Labels[LabelDefaultPreferenceKind]).To(Equal(LabelDefaultPreferenceKind))
+			Expect(pvc.Labels).To(HaveKeyWithValue(LabelDynamicCredentialSupport, "true"))
```

Comment:
Again could you replace the above assertions with `HaveKeyWithValue`?

Reply:
Done